invoicer_project/static/js/invoices_section_translation.js

```diff
@@ -0,0 +1,201 @@
+i18next.init({
+    lng: navigator.language,
+    fallbackLng: 'en',
+    resources: {
+        en: {
```

Comment:
Why don't you get translation from backend side? I mean you could send request to some endpoint and get translation